app/src/main/java/com/example/pigkeeper/NewPlayersActivity.kt

```diff
@@ -1,11 +1,34 @@
 package com.example.pigkeeper
 
-import androidx.appcompat.app.AppCompatActivity
+import android.content.Intent
 import android.os.Bundle
+import android.widget.Button
+import android.widget.EditText
+import androidx.appcompat.app.AppCompatActivity
+
 
 class NewPlayersActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_new_players)
+
+        val namesArray = ArrayList<String>()
+
+        val buttonStartRound = findViewById<Button>(R.id.buttonStartRound)
+        buttonStartRound.setOnClickListener{
+            println("here")
+            for(name in namesArray){
+                println(name)
+            }
+            startActivity(Intent(this@NewPlayersActivity, TurnOrderRollActivity::class.java))
+        }
+
+        val inputName = findViewById<EditText>(R.id.inputName)
+
```

Comment:
have to click button again to save last player, counterintuitive, maybe rename button to save player?